presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/ResourceGroupSpec.java

```diff
@@ -48,6 +49,7 @@
     private final Optional<Boolean> jmxExport;
     private final Optional<Duration> softCpuLimit;
     private final Optional<Duration> hardCpuLimit;
+    private final Optional<ResourceGroupQueryLimits> perQueryLimits;
```

Comment:
After thinking a bit - having this as a flat structure in this class as well will probably be a better way forward. That will just blend better with how all other properties are laid out